docs/contribution/release.md

```diff
@@ -0,0 +1,78 @@
+# how release works
+
+Prerequisite:
+
+* docker installed
+* How drone works
+* How [goreleaser](https://goreleaser.com/intro/) works
+* multi arch TODO, don't know yet
+
+Briefly at least.
+
+## Continuous Delivery
+
+We use drone as continuous integration and continuous delivery pipeline.
+Integration means: running tests, checking code quality.
+Delivery means: make a release when a new tag get pushed.
```

Comment:
We might want to add something here about delivery vs release. Whenever something gets merged into master, we push a new image tagged with that hash, e.g. `docker.io/packethost/cluster-api-provider-packet:abcde566`. When we cut a release, i.e. a git tag, we also push an image tagged with that tag, e.g. `docker.io/packethost/cluster-api-provider-packet:v0.3.1`

Reply:
Will do! Thanks, I didn't notice that